recipes/wxpython/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "wxPython" %}
+{% set version = "3.0.2.0" %}
+{% set sha256 = "d54129e5fbea4fb8091c87b2980760b72c22a386cb3b9dd2eebc928ef5e8df61" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+   fn: wxPython-src-{{ version }}.tar.bz2  # [not win]
+   url: http://sourceforge.net/projects/wxpython/files/wxPython/{{ version }}/wxPython-src-{{ version }}.tar.bz2  # [not win]
+   sha256: {{ sha256 }}  # [not win]
+   fn: {{ version }}.zip  # [win]
+   url: https://github.com/NOAA-ORR-ERD/wxpython_wheels/archive/{{ version }}.zip  # [win]
+   sha256: 3b40fad724172cd7186af95224ae5bf45262c405aac62f8441fe24f0a7a91429  # [win]
+
+build:
+  number: 2 # build is set to 2, as there are others in the wild.
+  # using these selectors to run different build scripts doesn't seem to work anymore :-(
```

Comment:
well, I guess it doesn't work anyway -- though if someone does pick this up, better to have the notes right there with the recipe -- rather than having to go look in issues.

Anyone know WHY those selectors don't work for choosing a build script -- there really are times when it's easier to separate the linux and os-x build scripts...